pkg/static/login.js

```diff
@@ -221,11 +246,17 @@
                req("textContent", document) &&
                req("replaceAll", String.prototype) &&
                req("finally", Promise.prototype) &&
-               req("supports", window.CSS) &&
-               css("display", "flex") &&
-               css("display", "grid") &&
-               css("selector(test)") &&
-               css("selector(:is(*):where(*))");
+               req("supports", window.CSS);
+
+        if (hard_req) {
+            css("display", "flex") &&
+            css("display", "grid") &&
+            css("selector(test)") &&
+            css("selector(:is(*):where(*))");
+            return true;
+        } else {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17774-20221017-161558-ae4aa609-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)